software/nosql/src/main/java/brooklyn/entity/nosql/riak/RiakNodeSshDriver.java

```diff
@@ -68,7 +77,7 @@ public RiakNodeImpl getEntity() {
     @Override
     public Map<String, String> getShellEnvironment() {
         MutableMap<String, String> result = MutableMap.copyOf(super.getShellEnvironment());
-        // how to change epmd port, according to 
+        // how to change epmd port, according to
         // http://serverfault.com/questions/582787/how-to-change-listening-interface-of-rabbitmqs-epmd-port-4369
         if (getEntity().getEpmdListenerPort() != null) {
             result.put("ERL_EPMD_PORT", "" + Integer.toString(getEntity().getEpmdListenerPort()));
```

Comment:
Can you fix this so it doesn't have the `"" + ` part before the `Integer#toString(Integer)` call?